### DISCUSSION TITLE
Trickshot Aimbot

### DISCUSSION THREAD

**Discussion by AsiaGuy** [IceHacks/SurvivCheatInjector#750] [Ideas]
Add trickshot aimbot to reflectable object. This might be hard but try to do it

**Comment by iBLiSSIN** [comment]
> Add trickshot aimbot to reflectable object. This might be hard but try to do it

Not just hard, it ‘s impossible

**Comment by AsiaGuy** [comment]
can you tell me why?

**Comment by iBLiSSIN** [comment]
> can you tell me why?

Because there isn't any survivio source code to indicate where to hit the reflective object to hit the enemy.  If you don't close this issue, IceHacks will close it soon because it's impossible

**Comment by IceHacks** [comment]
Possible. Also possible to make AimBot not shoot at reflective material.

**Comment by iBLiSSIN** [comment]
> Possible. Also possible to make AimBot not shoot at reflective material.

Hmm, are you sure about that

**Comment by AsiaGuy** [comment]
yay, thank you very much!

**Comment by iBLiSSIN** [comment]
> yay, thank you very much!

congratulations

**Comment by AsiaGuy** [comment]
If possible please add my idea to *To Do* list
